### ISSUE TITLE
flask-mqtt can't run on ip other than 127.0.0.1:5000 even though host = '0.0.0.0'

### ISSUE THREAD

**Issue by andreas2023** [stlehmann/Flask-MQTT#123]
when I try the example at https://github.com/stlehmann/Flask-MQTT/tree/master/example , when I access it via 127.0.0.1:5000 it works fine to subscribe / publish but when I change to using ip on the same PC at 192.168.1.70:5000 the program can't run either to subscribe / publish.
Why can this happen and how to deal with it?
I have changed the host to '0,0,0,0' but the result is still the same , 
the webpage can be displayed but no messages are displayed in the messages field of subscribed mqtt messages

Thanks in advance

**Comment by Sohaib90** [comment]
Can you elaborate more by sharing image snippets of the behavior you are expecting?
I get the gist of your problem, but usually it is better to discuss an issue with error traces or in this case image snippets of the behavior

**Comment by Sohaib90** [comment]
Let me explain briefly about the IP addresses being used here (what I understand about the paradigm)

When you run a server on your machine, it listens for incoming connections on a specific IP address and port. The IP address is the unique numerical label assigned to each device connected to a computer network that uses the Internet Protocol for communication. The two IP addresses you mentioned, `'0.0.0.0'` and `'127.0.0.1'`, have special meanings in the
context of server binding.` '0.0.0.0'` is a special IP address called the "wildcard" or "any" address. When a server binds to this
address, it means that the server should listen for incoming connections on all available network interfaces. This is useful if you want to make the server accessible from other machines on the network, but it also makes the server accessible from any IP address, including potentially malicious ones.

`'127.0.0.1'` is the loopback address. It is used to establish an IP connection to the same machine or computer being used by the end-user. When a server binds to this address, it means that the server can only be accessed from the local machine, which is useful for development and testing. On the other hand, IP addresses like `'192.168.1.1'` are private IP addresses assigned to a device, mostly routers, and cannot be bound to as they are not assigned to the machine you are trying to run the server on.

In summary, when you run a server using `socketio.run() `with the default host of  `127.0.0.1`, the server can only be accessed from the local machine, and if you want to make the server accessible from other machines on the network, you need to bind the server to the host's IP address or the special address `"0.0.0.0"` which means that the server should listen on all available
network interfaces.

**Comment by Sohaib90** [comment]
@andreas2023 since there is no additional problems reported I will close this issue
